O/OpenCV/build_tarballs.jl

```diff
@@ -67,7 +61,8 @@ fi
 
 cmake -DCMAKE_FIND_ROOT_PATH=${prefix} \
       -DJulia_PREFIX=${prefix} \
-      -DWITH_JULIA=ON \
+      -DWITH_JULIA=ON -DJulia_FOUND=ON -DHAVE_JULIA=ON -DJulia_WORD_SIZE=${nbits} \
```

Comment:
I recommend putting each option on its own line, that makes future changes to this nicer (the diffs will be more readable). It also makes it easier to spot when options are used multiple times by accident, as it is the case here:
```suggestion
      -DWITH_JULIA=ON \
      -DJulia_FOUND=ON \
      -DHAVE_JULIA=ON \
      -DJulia_WORD_SIZE=${nbits} \
```
so we can simplify to
```suggestion
      -DWITH_JULIA=ON \
      -DJulia_FOUND=ON \
      -DJulia_WORD_SIZE=${nbits} \
```